modules/nested-accordion/assets/js/frontend/handlers/nested-accordion-title-keyboard-handler.js

```diff
@@ -0,0 +1,64 @@
+import NestedTitleKeyboardHandler
+	from 'elementor-assets-js/frontend/handlers/accessibility/nested-title-keyboard-handler';
+
+export default class NestedAccordionTitleKeyboardHandler extends NestedTitleKeyboardHandler {
+	__construct( ...args ) {
+		super.__construct( ...args );
+		const config = args[ 0 ];
+		this.toggleTitle = config.toggleTitle;
+	}
+
+	getDefaultSettings() {
+		const parentSettings = super.getDefaultSettings();
+
+		return {
+			...parentSettings,
+			selectors: {
+				itemTitle: '.e-n-accordion-item-title',
+				itemContainer: '.e-n-accordion-item > .e-con',
+			},
+			ariaAttributes: {
+				titleStateAttribute: 'aria-expanded',
+				activeTitleSelector: '[aria-expanded="true"]',
+			},
+			datasets: {
+				titleIndex: 'data-accordion-index',
+			},
+		};
+	}
+
+	handeTitleLinkEnterOrSpaceEvent( event ) {
+		this.toggleTitle( event );
+	}
+
+	handleContentElementEscapeEvents( event ) {
+		const detailsNode = this.findParentDetailsNode( event?.currentTarget ),
+			summaryNode = detailsNode?.querySelector( 'summary' );
+
+		if ( summaryNode ) {
+			event.currentTarget = summaryNode;
+
+			this.getActiveTitleElement().trigger( 'focus' );
+			this.toggleTitle( event );
+		}
+	}
+
+	handleTitleEscapeKeyEvents( event ) {
+		const detailsNode = event?.currentTarget.parentElement,
+			isOpen = detailsNode?.open;
+
+		if ( isOpen ) {
+			this.toggleTitle( event );
+		}
+	}
+
+	findParentDetailsNode( el ) {
```

Comment:
```suggestion
	findParentDetailsNode( element ) {
```

Reply:
I am not sure if we need findParentDetailsNode(). Possibly it is easier to just use 'closest()', without a while loop.